installer/database/migrations/migrate_0.5.9.7.sql

```
@@ -0,0 +1,277 @@
+    autori_id    INT             NOT NULL,
+    authority_id BIGINT UNSIGNED NOT NULL,
+    created_at   TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    PRIMARY KEY (autori_id, authority_id),
+    KEY idx_authority_autore (authority_id),
+    CONSTRAINT fk_aal_authority FOREIGN KEY (authority_id) REFERENCES authority_records(id) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- ─── Part 2: Idempotent column additions (for installs that had an older ────
+-- ensureSchema() snapshot without all columns)
+
+SET @c := (SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA=DATABASE() AND TABLE_NAME='archival_units' AND COLUMN_NAME='material_status');
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)